extra/3-magic-8-ball.js

```diff
@@ -59,7 +46,16 @@ function shakeBall() {
   This function should expect to be called with any value which was returned by the shakeBall function.
 */
 function checkAnswer(answer) {
-  //Write your code in here
+  for (const group of Object.keys(magicAnswer))
```

Comment:
Just a small note here. Please ensure that the indentation is correctly done (especially the brackets). It should be:
```
for(...) {
    //code here
}
```

Reply:
Prettier has been working fine . I will have a look. Thanks !

Reply:
You can refer to https://syllabus.codeyourfuture.io/guides/code-style-guide